truss/templates/server/common/termination_handler_middleware.py

```diff
@@ -0,0 +1,45 @@
+import asyncio
+import signal
+from typing import Callable
+
+from fastapi import Request
+
+DEFAULT_TERM_DELAY_SECS = 5.0
```

Comment:
why is the default so long here? I thought the idea was to make this termination process shorter?

Reply:
I've added a comment to explain a bit, but mostly it's because it's hard to know a good value for this. This delay starts after the last response is ready, but there may be more middleware layers after and then bytes need to be pushed to the client, we don't want to terminate before that. 5 secs is likely sufficient for that. Could be lower but hard to say what's the right number here. Overall 5 seconds is still much better than minutes needed right now.